azure_sdk/lib/profile_azure_stack/modules/compute_profile_module.rb

```diff
@@ -181,34 +181,32 @@ class ComputeClass
 
       def initialize(configurable, base_url=nil, options=nil)
         @configurable, @base_url, @options = configurable, base_url, options
-        client = Azure::ARM::Compute::Api_2017_03_30::ComputeManagementClient.new(configurable.credentials, base_url, options)
-        if(client.respond_to?(:subscription_id))
-          client.subscription_id = configurable.subscription_id
-        end
-        @availability_sets = client.availability_sets
-        @virtual_machine_extension_images = client.virtual_machine_extension_images
-        @virtual_machine_extensions = client.virtual_machine_extensions
-        @virtual_machine_images = client.virtual_machine_images
-        @usage_operations = client.usage_operations
-        @virtual_machine_sizes = client.virtual_machine_sizes
-        @images = client.images
-        @resource_skus = client.resource_skus
-        @virtual_machines = client.virtual_machines
-        @virtual_machine_scale_sets = client.virtual_machine_scale_sets
-        @virtual_machine_scale_set_extensions = client.virtual_machine_scale_set_extensions
-        @virtual_machine_scale_set_vms = client.virtual_machine_scale_set_vms
-        @disks = client.disks
-        @snapshots = client.snapshots
-        @virtual_machine_run_commands = client.virtual_machine_run_commands
+
+        client_0 = Azure::ARM::Compute::Api_2017_03_30::ComputeManagementClient.new(configurable.credentials, base_url, options)
```

Comment:
minor: Although this is internally used name `client_0` but it'd be great if we can have `client_2017_03_30` just for readability but not required as of today